src/mumble/MainWindow.cpp

```diff
@@ -66,6 +66,7 @@ MainWindow::MainWindow(QWidget *p) : QMainWindow(p) {
 	qiIconMuteSelf.addFile(QLatin1String("skin:muted_self.svg"));
 	qiIconMuteServer.addFile(QLatin1String("skin:muted_server.svg"));
 	qiIconMuteSuppressed.addFile(QLatin1String("skin:muted_suppressed.svg"));
+	qiIconMutePushToMute.addFile(QLatin1String("skin:muted_pushtomute.svg"));
```

Comment:
For this to work, you would need to add the icon to `mumble.qrc`: https://github.com/mumble-voip/mumble/blob/master/src/mumble/mumble.qrc